src/components/Button.tsx

```diff
@@ -0,0 +1,9 @@
+export function Button(){
```

Comment:
We forgot to update this component declaration to be ES6 as well. We can update the whole function to something like;

```typescript
const Button = () => {
 const sumOfTwo = (num1:number, num2:number) => alert(`The sum of ${num1} and ${num2} is ${num1+num2}`) 
  return <button onClick={() => sumOfTwo(5,4) }> Click me </button>
}
export default Button
```